website/versioned_docs/version-2.0.0-beta.14/deployment.mdx

```diff
@@ -354,11 +356,21 @@ jobs:
         with:
           node-version: 14.x
           cache: yarn
-      - name: Build website
-        working-directory: website
-        run: |
-          yarn install --frozen-lockfile
-          yarn build
+
+      - name: Get yarn cache
+        id: yarn-cache
+        run: echo "::set-output name=dir::$(yarn cache dir)"
+
+      - name: Cache dependencies
+        uses: actions/cache@v2
+        with:
+          path: ${{ steps.yarn-cache.outputs.dir }}
+          key: ${{ runner.os }}-prescrypto-docs-${{ hashFiles('**/yarn.lock') }}
+          restore-keys: |
+            ${{ runner.os }}-prescrypto-docs-
+
+      - run: yarn install --frozen-lockfile
+      - run: yarn build
```

Comment:
If you want these as seperate steps, give them names as well

Reply:
This is how the original author wrote the code, I assume the commands are sufficiently verbose to be like so.

<img width="934" alt="image" src="https://user-images.githubusercontent.com/8750702/152279307-d8fb7b6b-a2d6-4481-a956-dd7dadc1ff07.png">

Src: https://github.com/peaceiris/actions-gh-pages#%EF%B8%8F-docusaurus

Reply:
Doesn't matter how they did it. Explicit names should be preferred. I also wonder why you split these into two steps... If the original workflow wasn't problematic, there's no need to change it.

Reply:
The original workflow was problematic.

Reply:
Did you make sure there's a line break in your workflow config? Or did you miss that pipe `|` at the end of `run:`?